src/css/index.css

```diff
@@ -0,0 +1,218 @@
+body
+{
+    margin: 0px;
+    background-color: var(--theme-ui-colors-background,#0C0717);
+    color: var(--theme-ui-colors-text,white);
+    box-sizing: border-box;
+}
```

Comment:
CSS variables have been used but I am not able to find the file where they are declared. I'm assuming you have used SCSS or LESS. Include those files as well from the next iterations.